pkg/divert/istio/virtualservices.go

```diff
@@ -80,37 +80,42 @@ func (d *Driver) restoreDivertService(vs *istioV1beta1.VirtualService) *istioV1b
 
 func (d *Driver) translateDivertHost(vs *istioV1beta1.VirtualService) *istioV1beta1.VirtualService {
 	result := vs.DeepCopy()
+	labels.SetInMetadata(&result.ObjectMeta, model.DeployedByLabel, d.name)
+	labels.SetInMetadata(&result.ObjectMeta, model.OktetoAutoCreateAnnotation, "true")
```

Comment:
Why we set the auto create annotation now when translating the divert host?

Reply:
This was a bug. If the virtual service already exists in the dev namespace, we shouldn't modify it (other than injecting the header). `model.OktetoAutoCreateAnnotation` header means the virtual service was created by divert from another namespace. In that case, if the virtual service in the other namespace is updated, we need to update also the copy in the developer namespace

Reply:
Thanks for the explanation!